### ISSUE TITLE
intl-messageformat is egregiously out of date

### ISSUE THREAD

**Issue by cellog** [i18next/i18next-icu#30]
## 🐛 Bug Report

The current version of `intl-messageformat` is 9.4.6, and this package depends on `2.2.0`. This breaks all of the date skeleton formats described in https://formatjs.io/docs/intl-messageformat#datetime-skeleton

Would you accept a PR bumping the intl-messageformat version in package.json?

## To Reproduce

A [codesandbox](https://codesandbox.io/examples/package/i18next) example or similar
or at least steps to reproduce the behavior:

```js
t("month-label", "{ date, date, ::MMMMyyyy }", { date: new Date() });
```

## Expected behavior

It should format with "February 2021" but instead formats as "02/08/2021"

## Your Environment

- *runtime version*: n/a
- *i18next version*: 19.8.4
- *os*: Mac
- *any other relevant information*

**Comment by jamuhl** [comment]
Sure PR's are very welcome - not using this myself and lacking time that would help a lot.

**Comment by jamuhl** [comment]
merged and published in i18next-icu@2.0.0
